### DISCUSSION TITLE
Issue/PR labels

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#953] [General]
Thanks @danflapjax for proactively labeling our issues! I've updated the Issue templates to apply those labels automatically when the templates are used. Let me know if anyone sees any issues.

**Comment by danflapjax** [comment]
You're welcome! I figured it would be fine to try adding labels, since if it turns out we don't like it it's very easy to revert by deleting the tags. The number of issues had gotten large enough that I wanted to make them easier to navigate; I hope other people will find them useful.
